Makefile.inc1

```diff
@@ -2049,11 +2049,6 @@ BRANCH_EXT_FROM=	rc
 .elif ${PKG_VERSION_FROM:M*.a*}
```

Comment:
This now happens even when PKG_VERSION_FROM might not be defined, though I suppose we might know it's always defined for other reasons I'm just not seeing. Do you know which it is?

Reply:
Funny enough, because of Github's interface, I think I saw the same thing you saw after you wrote this comment :) Once you expand the collapsed hunk, this comment doesn't seem to apply.

Reply:
Yea. You're right. Don't know what i was thinking